presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -361,10 +361,29 @@ public void testRoundTrip(Type type, List<?> readValues, TupleDomainFilter... fi
     public void testRoundTrip(Type type, List<?> readValues, List<Map<Subfield, TupleDomainFilter>> filters)
             throws Exception
     {
-        List<Map<Integer, Map<Subfield, TupleDomainFilter>>> columnFilters = filters.stream().map(filter -> ImmutableMap.of(0, filter)).collect(toImmutableList());
+        testRoundTrip(type, readValues, filters, filters);
+    }
+
+    /**
+     *
+     * @param type          Presto data type for the readValues.
+     * @param readValues    The values to be written, read and compared against.
+     * @param columnFilters The filters for the readers. For CHAR(n) values, the columnFilters should remove the paddings.
+     * @param valuesFilters The filters for the readValues. For CHAR(n) values, valuesFilters keeps the original paddings.
+     * @throws Exception
+     */
+    public void testRoundTrip(
+            Type type,
+            List<?> readValues,
+            List<Map<Subfield, TupleDomainFilter>> columnFilters,
+            List<Map<Subfield, TupleDomainFilter>> valuesFilters)
+            throws Exception
+    {
+        List<Map<Integer, Map<Subfield, TupleDomainFilter>>> streamReaderFilters = columnFilters.stream().map(filter -> ImmutableMap.of(0, filter)).collect(toImmutableList());
```

Comment:
Please reformat all stream operations and put each stream operation on a new line like this:
```
collection.stream()
  .op()
  .op()
```